CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unrealeased]
 
+## Fixed
+
+- Fix a bug in the learner dashboard contract page where an alert 
+  message was displayed on successful download.
+
```

Comment:
Take care during the rebase to put this entry in the Unrealeased section if there is a conflict